frontend/src/styles/QuestionNumberPage.module.scss

```diff
@@ -0,0 +1,85 @@
+.toporange{
+    position: absolute;
+    width: 70vh;
+    height: 70vh;
```

Comment:
These properties and transform and border-radius are repeated in all the blocks so you can group them up together in 1 CSS class and combine it with a more specific class for each block in QuestionNumberPage.jsx